DigitalLearningSolutions.Web/Helpers/TrackingSystem/Centre/CentrePage.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers.TrackingSystem.Centre

```

Comment:
Ah, sorry I just realised that this is under Helpers, but we should really add it to Models (and maybe even create enums folder)

Reply:
So for clarity, it should be under Web/Enums/TrackingSystem/Centre etc.?

Reply:
Oh so I meant more under Web/Models or potentially even Web/Models/Enums -- I guess we can have the TrackingSystem folder, but I don't think we'll need the Centre etc because we won't have that many models/enums in the web project. :)